templates/apps/srd-compendium.hbs

```diff
@@ -0,0 +1,33 @@
+<article>
+  <aside>
+    <header>
+      <h2>{{localize "DND5E.Disclaimer"}}</h2>
+    </header>
+    <section>{{{disclaimer}}}</section>
+  </aside>
+  <main>
```

Comment:
I believe you are not supposed to have more than one `<main>` element in an HTML document: https://html.spec.whatwg.org/multipage/grouping-content.html#the-main-element

I think we should perhaps just use `<div>` instead since the reason for this element container is just for layout.